META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced